test/data/search/8bins23window1error50pattern1overlap.out

```diff
@@ -0,0 +1,32 @@
+read-0	0,3,4,6,
+read-1	0,2,4,7,
+read-2	0,1,
+read-3	1,2,5,6,7,
+read-4	1,2,3,4,
+read-5	0,2,
+read-6	0,3,4,7,
+read-7	3,4,5,7,
+read-8	0,2,3,4,5,
+read-9	0,1,2,4,5,7,
+read-10	0,4,5,7,
+read-11	2,3,4,5,
+read-12	3,4,6,7,
+read-13	3,4,5,6,
+read-14	3,5,7,
```

Comment:
(23,19)-minimisers and probabilistic thresholding

Finds true positives and quite many false positives